lib/mappers/adobeXD/border.js

```diff
@@ -0,0 +1,26 @@
+const camelcase = require('camelcase');
+
+const { tokenize, normalizeColorValues, formatColor } = require('../../utils');
+
+function getRadius(shape) {
+  if (Object.prototype.hasOwnProperty.call(shape, 'r')) {
+    return shape.r;
+  } 
+
+  return 0;
+};
+
+function mapBorder(artboard) {
+  const { style } = artboard.value;
+  
+  return {
+    id: camelcase(artboard.name),
+    token: tokenize('border'),
+    width: style.stroke.width,
+    // type: style.stroke.align.toUpperCase(),
```

Comment:
This property doesn't exist on every border

Reply:
but that doesn't mean that you should skip it when it does?
Just set it to undefined if it doesn't exist on every border.

Also think it's weird it doesn't exist on every border since each border should be one of inside, center, outside right?

Reply:
I’ll investigate if it’s a bug in the xd file, intended behavior or an error on my part. For now it maps to null if it doesn’t exist.

Reply:
It seems that a center border is the default(?) and does not get specifically mapped. For now, we can read the property for outside and inside borders. When the property does not exist we map it to center.